CHANGELOG.asciidoc

```diff
@@ -652,13 +652,6 @@ https://github.com/elastic/beats/compare/v1.2.0...v5.0.0-alpha1[View commits]
 - Add `event_id`, `level`, and `provider` configuration options for filtering events {pull}1218[1218]
 - Add `include_xml` configuration option for including the raw XML with the event {pull}1218[1218]
 
-==== Known issues
```

Comment:
@tsg We had "known issues" in one of the past release. I think it is an interesting question if that is something we will need in the future also as part of the CHANGELOG?